docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -36,17 +33,36 @@ Prerequisites
 Software
 ~~~~~~~~
 
-- A web browser,
-- `Vagrant`_ (1.6.2 or newer),
-- `VirtualBox`_,
-- ``vagrant-scp`` plugin:
+All Platforms
+*************
+
+`virtualenvwrapper <https://virtualenvwrapper.readthedocs.org/en/latest/install.html>`_
+
+OS X
+*****
+
+`Homebrew <http://brew.sh>`_
+
+.. prompt:: bash $
+
+   brew tap stepanstipl/noop
+   brew install createrepo dpkg
+
+Ubuntu
+******
 
-  .. prompt:: bash $
+.. prompt:: bash $
 
-     vagrant plugin install vagrant-scp
+   apt-get update
```

Comment:
Ah thanks, I was doing this in a Docker container and didn't need that but I guess that isn't the normal case.